build.sh

```diff
@@ -65,14 +68,45 @@ then
   exit 1
 fi
 
+###########################################################################################################
```

Comment:
By embedding the processing of operator versions here as part of build.sh it means that the AMI will contain the operator versions available at the time of build, and that rebuilding an AMI version could contain different operator versions. In the future we could separate this processing into a separate pipeline job that produces an imageset-config artifact, and then pass a specific imageset-config artifact as a parameter to the AMI build. Something to think about for the future.